cloudera-on-centos/data-node.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json#",
```

Comment:
This should be http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json# for all the template files (parameter file has the correct schema)